synth.py

```diff
@@ -29,6 +29,19 @@
   library = gapic.node_library(name, version)
   s.copy(library, excludes=["package.json", "README.md"])
 
+# Expose helper module
+index_file = 'src/index.ts'
```

Comment:
So here I always wonder if we should do this or just exclude `src/index.ts` from the generation (line 30) and just have `src/index.ts` unchanged by the synthtool. Up to you here :)

Reply:
I think I'm going to err on the side of patching the library after generation, for now. We're doing some patching after the fact in the Python library too. @bcoe , your input gratefully accepted.

Reply:
If I'm following this regex, we're simply matching the end of the file? I think I like this approach for now 👍 as it allows us to add new clients via automation, without manual intervention -- let's try it.

Reply:
Awesome, thank you! I'm happy to work with you on potential better solutions in the future.

Reply:
Per conversation over chat, we decided to not do any post-processing in synth.py and instead exclude the hand-written files from being overwritten.